pallets/committee-management/src/impls.rs

```diff
@@ -423,6 +437,63 @@ impl<T: Config> Pallet<T> {
             Self::deposit_event(Event::BanValidators(fresh_bans));
         }
     }
+
+    /// Calculates committee for the given session.
+    /// If the era `E` starts in the session `a`, and ends in session `b` then from session `a-1`
+    /// to session `b-1` this function can answer question who will be in the committee in the era `E`.
+    /// In the last session of the era `E` this can be used to determine all of the sessions in the
+    /// era `E+1`.
+    pub fn session_committee_for_session(
+        session: SessionIndex,
+    ) -> Result<SessionCommittee<T::AccountId>, SessionValidatorError> {
+        let ae = match T::EraInfoProvider::active_era() {
```

Comment:
I think there are some issues in this function related to our lovely off-by-ones :fearful: In particular it is confusing to me why we need to look what's the `current_session` at all. (I might be wrong here, if so then I'm sorry :|  I know it's painful to think about these things all the time :P)
Anyway, my understanding is that `current_era_validators()` is synchronized with `current_era()` and not `active_era()`. This means that if we query `ce = T::EraInfoProvider::current_era()` then compute `ce_session_start` and `ce_session_end` then we can just check:
- `ce_session_start<= session <= ce_session_end`
and if the above holds, then `Self::rotate_committee_inner(&era_validators, committee_seats, session)` should output the correct result (with `era_validators` and `committee_seats` computed as below).

Reply:
We need to look at what current session is because we can only answer query for lets say era N in sessions `SS -1` to `ES - 1` where `SS` starts era `N` and `ES` ends it.

Reply:
But the solution with CurrentEra instead is cleaner. I will go with it

Reply:
Reworked :)